static/templates/template/container.html

```diff
@@ -1,8 +1,8 @@
-<md-subheader class="md-no-sticky">Use our Task Builder to design your task. Arrange and customize your task using our draggable template items. Double click a component to modify its properties.</md-subheader>
+<md-subheader class="md-no-sticky" ng-init="project.getLastMilestoneComments()">Use our Task Builder to design your task. Arrange and customize your task using our draggable template items. Double click a component to modify its properties.</md-subheader>
```

Comment:
@mbernst this makes it so that we don't load comments unless a person clicks on the tab. 
@shirish we can show the number of comments in the heading if we load it in the controller, I think that will be more helpful to the user.